ms/MSOper/MSMetaData.cc

```diff
@@ -3857,6 +3907,23 @@ MSMetaData::_getChunkSubScanProperties(
     return make_pair(scanProps, mysubscans);
 }
 
+void MSMetaData::_modifyFirstExposureTimeIfNecessary(
+    FirstExposureTimeMap& current, Int dataDescID,
+    Double time, Double exposure, const Unit& eunit
```

Comment:
please move this overload next to the other one in the file
you could probably also implement the Map, Map overload in terms of this one to remove a little duplicated logic.